src/sentry/utils/snuba.py

```diff
@@ -1329,7 +1340,7 @@ def aliased_query_params(
         raise ValueError("A dataset is required, and is no longer automatically detected.")
 
     derived_columns = []
-    resolve_func = resolve_column(dataset)
+    resolve_func: Callable = resolve_column(dataset)
```

Comment:
rather than this, the type annotation for `column_resolver` should return a Callable